middleware/authenticateUser.js

```diff
@@ -1,26 +1,55 @@
-const { auth } = require("../firebase");
+const jwt = require("jsonwebtoken");
```

Comment:
it should make use of the `JWT/index.js` instead of re doing the same flow again